.github/workflows/scale-test-100-gce.yaml

```diff
@@ -160,14 +161,16 @@ jobs:
 
       - name: Deploy cluster
         id: deploy-cluster
-        uses: cilium/scale-tests-action/create-cluster@238d773bd07754bfd693a6b22c94eddf3a12778d # main
+        uses: thorn3r/scale-tests-action/create-cluster@a9f91acd923775b8b90fa23fb1743ddc8b3efc03  # updates for 5k
         timeout-minutes: 30
         with:
           cluster_name: ${{ steps.vars.outputs.cluster_name }}
-          control_plane_size: n1-standard-8
-          control_plane_count: 1
+          control_plane_size: n2-standard-64
+          control_plane_count: 3
+          control_plane_volume_size: 50
           node_size: e2-medium
-          node_count: 100
+          node_count: 20
+          node_volume_size: 50
```

Comment:
`node_size: e2-standard-8` for heavy-deployment will need to be `e2-standard-32` probably:
https://github.com/kubernetes/test-infra/blob/8c94170825a87c6b6eb339a117263f6a214d0872/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L99C37-L99C51